src/socketHandler/gameHandler.js

```diff
@@ -93,11 +93,15 @@ const cardPlayed = async function (socket, io, payload) {
     io.to(lobbyId).emit('score', getPlayersScores(lobbyId));
 
     // If it is, start a new round
-    addRound(lobbyId);
-    addSubround(lobbyId);
-    const nextRound = getRounds(lobbyId).length;
-    const gameData = startRound(nextRound, players.length);
-    io.to(lobbyId).emit('startRound', gameData);
+    if (getRounds(lobbyId).length + 1 > lobby.maxRounds) {
+      io.to(lobbyId).emit('endGame', getPlayersScores(lobbyId));
+    } else {
```

Comment:
Minor, may use a return; in the if instead of the else. just as we used in the other if's

Personally, I would revert the if so that the endGame call is at the end